content_schemas/formats/shared/definitions/_organisation_logo.jsonnet

```diff
@@ -10,7 +10,6 @@
     crest: {
       type: [
         "string",
-        "null",
       ],
```

Comment:
```suggestion
      type: "string"
```

minor, you don't need to define this in an array.

Reply:
Thanks.  I'll fix this.